renderers/scoreboard.py

```diff
@@ -29,3 +30,4 @@ def render(self):
       PitchesRenderer(self.canvas, self.scoreboard.pitches, self.data).render()
       OutsRenderer(self.canvas, self.scoreboard.outs, self.data).render()
       BasesRenderer(self.canvas, self.scoreboard.bases, self.data).render()
+    NetworkErrorRenderer(self.canvas, self.data).render()
```

Comment:
Can you move this up to the top of the render function? It's kind of hidden down here at the bottom below the conditional

Reply:
I put the network error renderer on the very end of each of the render methods so that it will always appear on top of everything else. The way we do things now, if I put it on top there's a chance other elements will completely cover it up.